README.md

```diff
@@ -20,27 +20,81 @@ This repository has been created to coordinate developers' work in comparing Ope
 
 ## Development machine setup
 
-## Running an IIIF Image Server (Mac)
+This is a three-step process:
 
-This directory contains the Java files for a Cantaloupe Image Server and an `images` directory from which local images are served. 
+1. Run the image server:
```

Comment:
Thanks @ARBaynes - could you just add a note here to say that the server is required for the `manifest.json` and UV?